examples/model_diagnostics/model_diagnostics_demo.ipynb

```diff
@@ -616,6 +616,30 @@
     "        break\n",
     "    print(model_run_data_row.url)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a41f9f33",
+   "metadata": {},
+   "source": [
+    "### Export model run labels"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4573e2c7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MODEL_ID = ''\n",
+    "MODEL_RUN_ID = ''\n",
+    "model = client.get_model(MODEL_ID)\n",
+    "\n",
+    "def is_correct_model_run(run): return run[\"id\"] == MODEL_RUN_ID\n",
+    "model_run = next(filter(is_correct_model_run, model.model_runs), None)\n",
```

Comment:
It might be cleaner to use a lambda function:
`next(filter(lambda run: run['id'] == MODEL_RUN_ID, model.model_runs), None)`
Hopefully we can update the sdk to support this directly.